### ISSUE TITLE
I have a dataset with 7 classes, but i want to finetune and train my object detection model with only two classes. i still want the images with other 5 classes in training set so that my model can actually learn what to exclude. with other YOLOs images without label files works fine. but with YOLO_NAS, it excludes the images without having label files while training.

### ISSUE THREAD

**Issue by DarshnaZE** [Deci-AI/super-gradients#1330]
### 🐛 Describe the bug

Earlier I used the SG_transfer_learning_object_detection.ipynb for my approach to finetune YOLO-NAS. Then, I tried with  https://github.com/Deci-AI/super-gradients/blob/master/documentation/source/ObjectDetection.md#how-to-connect-your-own-dataset, but ended with the following error:                                          So i am training YOLO-NAS with with 640*640 or 960*544 (multiple of 32) with large backbone. I have dataset with 165552 images in my train set, but only 143769 objects have their label files. other images (21783) do not have their label files. while caching annotations 165552 are getting cached. But during training per iteration only (4876*32) 156032 are being considered. Not sure what's happening with the remaining objects/images. This is the same issue, I was facing with my earlier approach as well. Is ther any way i can track the numbers/metrics for each class during training and validation? i tried to use verbose, but didn't end up with getting metrics values for each class. can you please guide me, where can i set verbose flag?             ![image](https://github.com/Deci-AI/super-gradients/assets/101351723/767cbeb2-6270-437b-bc37-e66c5b7ecc81)
 

### Versions

[pip3] numpy==1.22.2
[pip3] pytorch-quantization==2.1.2
[pip3] torch==1.14.0a0+44dac51
[pip3] torch-tensorrt==1.4.0.dev0
[pip3] torchtext==0.13.0a0+fae8e8c
[pip3] torchvision==0.15.0a0
[pip3] triton==2.0.0

**Comment by BloodAxe** [comment]
Both `YoloDarknetFormatDetectionDataset` and `COCOFormatDetectionDataset` has `ignore_empty_annotations` argument.

If `ignore_empty_annotations=True` - images with no annotations will be excluded and not participate in training. 
If `ignore_empty_annotations=False` - all images will participate in training. Looks like this is the option you want to enable for your task.

**Comment by DarshnaZE** [comment]
> Hey @BloodAxe  thanks, Will test with that.

**Comment by hieunm1821** [comment]
I do not see `ignore_empty_annotations` in `YoloDarknetFormatDetectionDataset`. Set `ignore_empty_annotations` is False still ignore the empty label
